src/Patterns/Path.php

```diff
@@ -1,413 +1,145 @@
 <?php
```

Comment:
Although it is probably more in line with how nodes, relationships and paths are actually implemented in Cypher...

Reply:
It's mainly because I did not think of a better way. It also exactly maps to the way it is implemented in Bolt and the original HTTP implementations, as well as all the official drivers:

https://7687.org/packstream/packstream-specification-1.html#path---structure

Reply:
This is good